src/test/unit/EigenPodUnit.t.sol

```diff
@@ -157,6 +157,15 @@ contract EigenPodUnitTests is Test, ProofParsing {
         fuzzedAddressMapping[address(0x0)] = true;
     }
 
+    function testDeployPodWithStateChecks() external {
+        cheats.startPrank(address(42000094992494));
```

Comment:
Aren't we already testing this in `testCheckThatHasRestakedIsSetToTrue` ?

Reply:
This is checking it on a fresh deployed pod? Equivalent but this feels more contained as a unit test.

Reply:
ohh. just found it's already implemented in `testDeployedPodIsRestaked`

Reply:
Ah ok cool :)